randomizer/CollectibleLogicFiles/AngryAztec.py

```diff
@@ -143,7 +143,7 @@
     ],
     Regions.LlamaTempleBack: [
         Collectible(Collectibles.banana, Kongs.tiny, lambda l: True, None, 2),
-        Collectible(Collectibles.bunch, Kongs.tiny, lambda l: l.CanSlamSwitch(Levels.AngryAztec, 1) or l.twirl or l.advanced_platforming, None, 2),  # Behind Mini tunnel
+        Collectible(Collectibles.bunch, Kongs.tiny, lambda l: l.CanSlamSwitch(Levels.AngryAztec, 1) or (l.twirl or l.advanced_platforming), None, 2),  # Behind Mini tunnel
```

Comment:
This one implies you can get the 2 bunches without any moves if you have advanced platforming (twirl OR advanced platforming)